e2e/linodevpc-controller/minimal-linodevpc/chainsaw-test.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: chainsaw.kyverno.io/v1alpha1
 kind: Test
 metadata:
   name: minimal-linodevpc
+  # Label to trigger the test on every PR
+  labels:
+    quick:
+    min-vpc:
```

Comment:
I could go either way here but I sort of think we should maybe keep the label generic to the component rather than the test, so all tests that test VPC get a `vpc` label. what are your thoughts on that approach?